src/renderer/src/components/PasswordStrenght.tsx

```diff
@@ -0,0 +1,53 @@
+import { useTranslation } from 'react-i18next'
+import { regexGood, regexStrong } from '@renderer/constants/passwordStrenght'
+import { StyleHelper } from '@renderer/helpers/StyleHelper'
+
+type TProps = {
+  password: string
+}
+
+export const PasswordStrenght = ({ password }: TProps) => {
+  const { t } = useTranslation('components', { keyPrefix: 'passwordStrenght' })
+
+  return (
+    <div className="w-full mt-2.5">
+      <div className="flex w-full gap-1">
+        <div
+          className={StyleHelper.mergeStyles('w-1/3 bg-gray-300/70 rounded h-1', {
+            'bg-pink': password.length > 4 || regexGood.test(password) || regexStrong.test(password),
+          })}
```

Comment:
Create a const for these regex tests, so you can reuse it below instead of testing every time